src/pages/Finance.tsx

```diff
@@ -212,7 +212,7 @@ const Finance = () => {
     <div className="space-y-6">
       <HeaderCard
         title="Finance"
-        aside={<Link to="/governance/new-proposal" label="New transfer" />}
+        aside={<Link to="/finance/new-deposit" label="New deposit" />}
```

Comment:
I qas thinking this for this button:
Have it be a dropdown with 2 items, deposit and withdraw, where deposit takes you to deposit page and withdraw to new proposal page

Reasoning: I have an idea brewing in my head to make it possible to link the the new-proposal page with query params to fill some of the form content immediately. Not high priority, but might as well make it easily adaptable to this

Reply:
I've turned this into a seperate JIRA task, since this PR has already become quite large.